src/draco/tools/fuzz/draco_mesh_encoder_fuzzer.cc

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fuzzer/FuzzedDataProvider.h>
+
+#include "draco/src/draco/mesh/mesh.h"
+#include "draco/src/draco/mesh/triangle_soup_mesh_builder.h"
+#include "draco/src/draco/compression/encode.h"
+#include "draco/src/draco/compression/expert_encode.h"
+#include "draco/src/draco/core/encoder_buffer.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+
+  // Build the mesh
+  draco::TriangleSoupMeshBuilder mesh_builder;
+  FuzzedDataProvider stream(data, size);
+
+  const uint num_faces = 5;
```

Comment:
Should use int.
https://google.github.io/styleguide/cppguide.html#Integer_Types

Reply:
Overall I'm not sure how useful this will be if you only ever encode 5 faces.

Reply:
I had issues with the speed at which this particular fuzzer runs. The number of faces used to be a pseudorandom uint8_t from the FuzzedDataProvider stream, but that was far too slow. Any suggestions for improving this?